### PR TITLE
Fixes #61 Add clarifications to the animal rescue demo on K8s

### LINKED ISSUE #61: Add clarifications to the animal rescue demo on K8s

### PR DESCRIPTION
**Pull request by ddobrin**
Please review the PR and merge it if you consider that it adds more details to the existing doc

### CODE REVIEW

File: k8s/deploy_to_k8s.md [hunk 1]
```diff
@@ -4,11 +4,38 @@
...
+
+$ curl https://dev-1234567.okta.com/oauth2/abcd12345/.well-known/openid-configuration
+
+{
+  "issuer": "https://dev-1234567.okta.com/oauth2/abcd12345",
+...
+  "jwks_uri": "https://dev-1234567.okta.com/oauth2/abcd12345/v1/keys",
+....
+
+# Please note that the format used is jwks_uri="<issuer-uri>/v1/keys"
```
Comment (LittleBaiBai) [inline k8s/deploy_to_k8s.md:29]: This line might be confusing for users not using Okta. Could you add `format used by Okta` or remove this line?

**Review by LittleBaiBai — APPROVED** [review]
Thanks for your PR! This PR will make is so much easier to try animal-rescue on k8s!

File: k8s/deploy_to_k8s.md (hunk 1 above)
Comment (ddobrin) [reply k8s/deploy_to_k8s.md:29]: yes, one minute

Comment (ddobrin) [reply k8s/deploy_to_k8s.md]: done

**Review by csterwa — APPROVED** [review]
Great improvements! Thank you!

**Pull request merged**